### ISSUE TITLE
Vertical page scrolling not working on Ipad

### ISSUE THREAD

**Issue by StefanDV** [pisi/Reel#149]
Hi,

First of all, great plugin. We are very happy with it, it does exactly what we want.

Except for one tiny problem, scrolling the page vertically does not seem to work. When I drag vertically over an image the page will not scroll. I tried it on you first example (F1 car) (http://jquery.vostrel.cz/reel#demo) and it won't scroll either.

You had this problem before and fixed it, but I can't get it to work.

My code:

``` js
$(".bike360Content img").reel(
    {
        frames: 24,
        preload: "linear",
        images: image##.jpg,
        cursor: "undefined",
        loops: false,
        cw: true,       
        wheelable: false
    });
```

Can you help me? Am i doing something wrong or is it a problem with the plugin?

Thanks,

Stefan

**Comment by pisi** [comment]
Add `scrollable: true` and scrolling on iOS will resume. This option has no effect for desktop browsers.

Thanks btw.! :)

**Comment by StefanDV** [comment]
Thank you for your reaction,

I already tried scrollable: true, but it does not seem to work, I can drag what I want, but the page will not scroll.

I have this problem on my iPad and iPhone, both running ios6, and on my work iPad running ios5.1.1.

I'm using jquery version 1.7.2.

Do you have another solution?

Btw, it is not a big problem, our website works fine and you can still scroll the page if you drag next to the reel, but It would be nice if it would work.

**Comment by pisi** [comment]
Oh, OK, then it is a bug after all. Thanks for reporting it!

**Comment by pisi** [comment]
@StefanDV, I believe I've fixed this issue. Please try out the code from the [development branch](/pisi/Reel/tree/development) if you can verify it being able to scroll the page by the Reel image on iPad. Should work even without the `scrollable: true` as this is the default value ;)

**Comment by StefanDV** [comment]
@pisi Great work man. It works. You fixed it real quick, didn't expect that. I tested it on the ipad and it scrolls like a charm.

We use reel in a website we made for a major bike manufacturer in The Netherlands, you can see it in the link below (launched yesterday).
For now it is only in Dutch, but international websites are coming.

http://www.batavus.nl/elektrischefietsen/Fuego-E-go-8P.htm 

Thanks again :) keep up the good work.

**Comment by pisi** [comment]
Thanks. As they say, your problems are my problems ;) I am glad people do find and consequently report back bugs. It is a huge help for me. And am very glad we resolved this one :) It will land in 1.2.1- I'll let you know then.

The website is very nice btw.!

Best,

**Comment by pisi** [comment]
Merged into [master](/pisi/Reel/tree/master) and **included in today's [version 1.2.1](/pisi/Reel/tree/v1.2.1)** release.
